packages/react-components/src/components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+
+import { ChevronDown } from '@livechat/design-system-icons/react/tabler';
+import cx from 'clsx';
+
+import { ActionMenu, ActionMenuItem } from '../ActionMenu';
+import { Icon } from '../Icon';
+
+import { ActionBarItem, IActionBarOption } from './ActionBarItem';
+
+import styles from './ActionBar.module.scss';
+
+export interface IActionBarProps {
+  /**
+   * The CSS class for menu container
+   */
+  className?: string;
+  /**
+   * The unique id key
+   */
+  id?: string;
+  /**
+   * Array of action bar options
+   */
+  options: IActionBarOption[];
+  /**
+   * Set the key for active element
+   */
+  activeOptionKey?: string | null;
```

Comment:
do we need this `Key` suffix?
```suggestion
  activeOption?: string | null;
```

Reply:
Name references to the part of the `option` object, so the `key` suffix is correct for me.